src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -24,28 +24,39 @@
 package io.zold.api;
 
 import java.io.IOException;
+import java.math.BigInteger;
 import java.time.ZonedDateTime;
 import java.util.regex.Pattern;
+import org.cactoos.scalar.IoCheckedScalar;
 import org.cactoos.scalar.ItemAt;
+import org.cactoos.text.FormattedText;
 import org.cactoos.text.SplitText;
 import org.cactoos.text.TextOf;
+import org.cactoos.text.UncheckedText;
 
 /**
  * RtTransaction.
  *
  * @since 0.1
+ * @checkstyle MagicNumber (500 lines)
```

Comment:
@carlosmiranda I don't think it is a good idea to have this in a non-test class: this is too broad and will create problems in the future.
It should be applied line by line everywhere it is needed.

Reply:
@victornoel to me it creates too much noise. Let's see what @llorllale thinks

Reply:
@carlosmiranda let's have it on a case-by-case basis. Having this suppressions create noise is a good thing because it brings attention to the fact that we're using magic numbers